operator/gitops/argocd/pipeline-service/openshift-pipelines/tekton-config.yaml

```diff
@@ -8,8 +8,6 @@ metadata:
     # the CRD will be applied and the resource can be created once the required dependencies are met.
     argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 spec:
-  addon:
-    enablePipelinesAsCode: false
```

Comment:
since PAC is no more an addon with OSP-1.9, we don't need this option in TektonConfig.